packages/cip2/src/selectionConstraints.ts

```diff
@@ -0,0 +1,51 @@
+import { CardanoSerializationLib, CSL, ProtocolParametersRequiredByWallet } from '@cardano-sdk/core';
+import { ComputeMinimumCoinQuantity, EstimateTxFee, SelectionSkeleton } from './types';
+
+export type BuildTx = (selection: SelectionSkeleton) => Promise<CSL.Transaction>;
+
+export const computeMinimumCost =
+  (
+    csl: CardanoSerializationLib,
+    { minFeeCoefficient, minFeeConstant }: ProtocolParametersRequiredByWallet,
+    buildTx: BuildTx
+  ): EstimateTxFee =>
+  async (selection) => {
+    const tx = await buildTx(selection);
+    return BigInt(
+      csl
+        .min_fee(
+          tx,
+          csl.LinearFee.new(
+            csl.BigNum.from_str(minFeeCoefficient.toString()),
+            csl.BigNum.from_str(minFeeConstant.toString())
+          )
+        )
+        .to_str()
+    );
+  };
+
+//
+export const computeMinimumCoinQuantity =
+  (
+    csl: CardanoSerializationLib,
+    { coinsPerUtxoWord }: ProtocolParametersRequiredByWallet
+  ): ComputeMinimumCoinQuantity =>
+  (multiasset) => {
+    const minUTxOValue = CSL.BigNum.from_str((BigInt(coinsPerUtxoWord) * 29n).toString());
```

Comment:
```suggestion
    const minUTxOValue = CSL.BigNum.from_str((coinsPerUtxoWord * 29).toString());
```
Does this need to be a `bigint`?